linter.py

```diff
@@ -12,13 +12,17 @@
 
 import re
 from SublimeLinter.lint import Linter
+from SublimeLinter.lint import persist
 
 
 class JSHint(Linter):
 
     """Provides an interface to the jshint executable."""
 
-    syntax = ('javascript', 'html', 'javascriptnext')
+    default_list = ["html", "laravel-blade", "javascript", "javascriptnext"]
```

Comment:
I use also blade but I wouldn't add this as a default option, most of javascript devs simply doesn't use blade.